test/messaging.test.ts

```diff
@@ -0,0 +1,106 @@
+import { solidity } from "ethereum-waffle";
+import chai, { assert, expect } from "chai";
+import { simpleDeploy } from "@makerdao/hardhat-utils";
+import hre, { ethers, network, starknet } from "hardhat";
+import { StarknetContract, HardhatUserConfig } from "hardhat/types";
+import config from "../hardhat.config";
+
+describe("test contracts interaction", function () {
+
+    it("check deployment data", async () => {
+        const { l1Contract, l2Contract, l2user, mockStarknetMessagingAddress, admin, l1_user } = await setupTest();
+        const _selector = await l1Contract.SELECTOR();
+        expect(_selector).to.be.eq(BigInt(1447471469451581726846414018284835017311951023166934288557129659906920235665))
```

Comment:
you can import a function to compute selector using js: `import { getSelectorFromName } from "starknet/dist/utils/hash";`